dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/passwordreset/PasswordResetServiceBean.java

```diff
@@ -90,12 +96,36 @@ public PasswordResetInitResponse requestPasswordReset( BuiltinUser aUser, boolea
         
     }
 
+    private void updateTokenExpirationTime(BuiltinUser aUser) {
+
+        long ONE_MINUTE_IN_MILLISECONDS = 60000;
+        long defaultInMilliseconds = systemConfig.getMinutesUntilPasswordResetTokenExpires() * ONE_MINUTE_IN_MILLISECONDS;
+
+
+        List<PasswordResetData> passwordResetDatas = findPasswordResetDataByDataverseUser(aUser);
```

Comment:
You are trying to find entity which is not yet saved in the DB it will never find it. Also you just wanted to find one Object so getting list is unnecessary and confusing.